pyproject.toml

```diff
@@ -26,3 +26,5 @@ aiohttp-json-rpc = "0.11.2"
 ujson = "1.35"
 toml = "0.9.4"
 pysha3 = "1.0.2"
+ed25519 = "1.4.0"
```

Comment:
I would like to add a todo notice on migrating to `py-cryptography` implementation when it is available https://github.com/pyca/cryptography/pull/4114

Reply:
@eugene-babichenko will do